node/aas-aid/test/asset-interfaces-description-test.ts

```diff
@@ -1327,4 +1327,90 @@ class AssetInterfaceDescriptionTest {
         expect(isValid.valid, isValid.errors).to.equal(true);
         expect(submodel).to.have.property("submodelElements").to.be.an("array").to.have.lengthOf(1);
     }
+
+    td4: ThingDescription = {
+        "@context": [
+            "https://www.w3.org/2022/wot/td/v1.1",
+            {
+                myPrefix1: "https://example.com/myContext1",
+                myPrefix2: "https://example.com/myContext2",
+            },
+        ],
+        title: "test thing",
+        "@type": ["myPrefix1:suffix1", "myPrefix2:suffix2"],
+        securityDefinitions: {
+            nosec_sc: {
+                scheme: "nosec",
+            },
+        },
+        base: "http://example.com:3003",
+        security: ["nosec_sc"],
+        "myPrefix2:suffix3": "value1",
+        properties: {
+            myProperty1: {
+                type: "number",
+                title: "Property Title",
+                "myPrefix1:suffix4": "myPrefix2:value4",
```

Comment:
I don't really understand the idea/system/schema behind the attached numbers but it is fine by me